landing-page/layouts/_default/single.html

```diff
@@ -2,11 +2,25 @@
 <head>
     <link href="{{ .Site.BaseURL }}/css/markdown.css" rel="stylesheet">
     <link href="{{ .Site.BaseURL }}/css/katex.min.css" rel="stylesheet">
+    <script>
+        function addAnchor(element) {
+            element.insertAdjacentHTML('beforeend', `<a href="#${element.id}" class="anchortag" ariaLabel="Anchor"> 🔗 </a>` )
+        }
+        document.addEventListener('DOMContentLoaded', function () {
+            var headers = document.querySelectorAll('h1[id], h2[id], h3[id], h4[id]')
+            if (headers) {
+                headers.forEach(addAnchor)
+            }
+        });
```

Comment:
Should we maybe throw a comment on here to remind us what this does? The code is relatively clear, but the querySelectorAll has that hard coded `h1[id] … h4[id]`. Also, will we need to update this if / when we add another header level?

Reply:
I added a comment describing what this does. I think h1-h4 is common for anchor tags since people usually don't need an anchor tag link to drill all the way down to an h5. (We could probably get away with h1-h3)